frontend/integration-tests/views/crud.view.ts

```diff
@@ -12,6 +12,11 @@ export const createYAMLLink = $('#yaml-link');
 export const saveChangesBtn = $('#save-changes');
 export const reloadBtn = $('#reload-object');
 export const cancelBtn = $('#cancel');
+export const addStorageItem = element(by.cssContainingText('.dropdown li a', 'Add Storage'));
```

Comment:
We have helpers now for actions in `crud.view.ts`. We should prefer those.

Reply:
Great. Reused clickDetailsPageAction()